src/App.tsx

```diff
@@ -17,16 +18,50 @@ export interface AppProps {
 }
 
 const App = (props: AppProps) => {
-    const informasjonsboksAlternativer = ['CV', 'Jobbønsker', 'Oppfølging', 'Personalia', 'Registrering', 'Ytelser'];
+    const overblikkFilter = useOverblikkFilter();
+
+    const informasjonsboksAlternativer: string[] = useMemo(
+        () => ['CV', 'Jobbønsker', 'Oppfølging', 'Personalia', 'Registrering', 'Ytelser'],
+        []
+    );
 
     const [valgteInformasjonsbokser, setValgteInformasjonsbokser] = useState<string[]>(informasjonsboksAlternativer);
+    const [avmerketInformasjonsbokser, setAvmerketInformasjonsbokser] = useState<string[]>([]);
+
+    const [lagredeInformasjonsbokser, setLagredeInformasjonsbokser] = useState<string[]>([]);
+
+    const setInitielState = useCallback(
+        (informasjonbokser: string[]) => {
+            setValgteInformasjonsbokser(informasjonbokser);
+            setAvmerketInformasjonsbokser(informasjonsboksAlternativer.filter((x) => !informasjonbokser.includes(x)));
+        },
+        [informasjonsboksAlternativer]
+    );
+
+    const areEqual = (array1: string[], array2: string[]) => {
+        return JSON.stringify(array1) === JSON.stringify(array2);
```

Comment:
I don't think `JSON.stringify` guarantees the same output given the same input, that is it does not guarantee that the order of the keys in the output-string will be the same each time the function is invoked on the same data. So I'm not sure its a good idea to use it for comparison purposes.